Src/coffeescript/cql-execution/test/util/comparison.coffee

```diff
@@ -113,3 +113,6 @@ describe 'equivalent', ->
     equivalent('123', '123').should.be.true()
     equivalent(123, 123).should.be.true()
     equivalent('123', new Code('123', 'test', '2016')).should.be.false()
+
+  it 'should consider codes with different versions to be equivalent', ->
```

Comment:
I think a test should be added to check equivalence between code and concept/code lists.

Reply:
Added some below.